django/test/testcases.py

```diff
@@ -614,6 +614,36 @@ def assertRaisesMessage(self, expected_exception, expected_message, *args, **kwa
         with cm:
             callable_obj(*args, **kwargs)
 
+    @contextmanager
+    def _assert_warns_message_cm(self, expected_warning, expected_message):
+        with self.assertWarns(expected_warning) as cm:
+            yield cm
+        self.assertEqual(str(cm.warning), expected_message)
```

Comment:
There's a slight difference from assertRaisesMessage here in that this uses assertEqual instead of assertIn. Is it intentional? Not that I necessarily disagree with the decision. I haven't looked at the reason for using assertIn in the first place. In either case, either the code should be changed or the docstring should be updated.

Reply:
It seemed more appropriate to do so this way. I'll update the method to ensure that the behavior of`assertWarnsMessage` is similar to `assertRaisesMessage`. 😉